api_docs/integrations-overview.md

```diff
@@ -1,47 +1,105 @@
 
-Zulip also makes it very easy to write your own integration, and (if you'd
-like) to get it merged into the main Zulip repository.
+Zulip offers [over 120 native integrations](/integrations/), which take
+advantage of Zulip's [topics](/help/introduction-to-topics) to organize
+notification messages. Additionally, thousands of integrations are available
+through [Zapier](https://zapier.com/apps) and [IFTTT](https://ifttt.com/search).
+You can also [connect any webhook designed to work with
+Slack](/integrations/doc/slack_incoming) to Zulip.
 
-Integrations are one of the most important parts of a group chat tool like
-Zulip, and we are committed to making integrating with Zulip as easy as
-possible.
+If you don't find an integration you need, you can:
 
-## Set up an existing integration
+- [Write your own integration](#write-your-own-integration). You can [submit a
+pull
+request](https://zulip.readthedocs.io/en/latest/contributing/reviewable-prs.html)
+to get your integration merged into the main Zulip repository.
 
-Most existing integrations send content from a third-party product into
-Zulip.
+- [File an issue](https://github.com/zulip/zulip/issues/new/choose) to request
+  an integration (if it's a nice-to-have).
 
-* Search Zulip's [list of native integrations](/integrations/) for the
-  third-party product. Each integration has a page describing how to set it
-  up.
+- [Contact Zulip Sales](mailto:sales@zulip.com) to inquire about a custom
+  development contract.
 
-* Check if [Zapier](https://zapier.com/apps) has an integration with the
-  product. If it does, follow [these instructions](/integrations/doc/zapier)
-  to set it up.
+## Set up an integration
 
-* Check if [IFTTT](https://ifttt.com/search) has an integration with the
-  product. If it does, follow [these instructions](/integrations/doc/ifttt)
-  to set it up.
+### Native integrations
 
-* Use a third-party webhook integration designed to work with
-  [Slack's webhook API](https://api.slack.com/messaging/webhooks)
-  pointed at Zulip's
-  [Slack-compatible webhook API](/integrations/doc/slack_incoming).
+{start_tabs}
 
-* If the product can send email notifications, you can
-  [send those emails to a channel](/help/message-a-channel-by-email).
+1. [Search Zulip's integrations](/integrations/) for the product you'd like to
+   connect to Zulip.
+
+1. Click on the card for the product, and follow the instructions on the page.
+
+{end_tabs}
+
+### Integrate via Zapier or IFTTT
+
+If you don't see a native Zulip integration, you can access thousands of
+additional integrations through [Zapier](https://zapier.com/apps) and
+[IFTTT](https://ifttt.com/search).
+
+{start_tabs}
+
+1. Search [Zapier](https://zapier.com/apps) or [IFTTT](https://ifttt.com/search)
+for the product you'd like to connect to Zulip.
+
+1. Follow the integration instructions for [Zapier](/integrations/doc/zapier) or
+   [IFTTT](/integrations/doc/ifttt).
+
+{end_tabs}
+
+### Migrate an integration from Slack
```

Comment:
"Migrate" is a little too limiting -- this feature is useful for the situation that you're already using Zulip and want to integrate with a third-party product that only offers a "Slack webhook" and not a "generic webhook".